tests/xrayignorerule_test.go

```diff
@@ -0,0 +1,54 @@
+package tests
+
+import (
+	"testing"
+	"time"
+
+	"github.com/jfrog/jfrog-client-go/xray/services/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestXrayIgnoreRule(t *testing.T) {
+	initXrayTest(t)
+	t.Run("createCveIgnoreRule", createCveIgnoreRule)
+}
+
+func deleteIgnoreRule(t *testing.T, ignoreRuleId string) {
+	err := testsXrayIgnoreRuleService.Delete(ignoreRuleId)
+	assert.NoError(t, err)
+}
+
+func createCveIgnoreRule(t *testing.T) {
+	var ignoreRuleId string
+	defer func() {
+		if ignoreRuleId != "" {
+			deleteIgnoreRule(t, ignoreRuleId)
+		}
+	}()
+
+	component := utils.IgnoreFilterNameVersion{
+		Name:    "gav://org.postgresql:postgresql",
+		Version: "42.2.3.jre7",
+	}
+	components := []utils.IgnoreFilterNameVersion{component}
+
+	cve := []string{"CVE-2022-31197"}
+	ignoreRuleFilter := utils.IgnoreFilters{
+		CVEs:       cve,
+		Components: components,
+	}
+
+	createIgnoreRule(t, &ignoreRuleId, ignoreRuleFilter)
```

Comment:
You are returning `*utils.IgnoreRuleParams` from `createIgnoreRule` but it is not used